crates/rspack_core/src/chunk_group.rs

```diff
@@ -356,15 +355,22 @@ impl EntryRuntime {
 
 // pub type EntryRuntime = String;
 
-#[derive(Debug, Default, Clone, Hash, PartialEq, Eq, PartialOrd, Ord)]
+#[derive(Derivative, Debug, Default, Clone)]
+#[derivative(Hash, PartialEq, Eq, PartialOrd, Ord)]
 pub struct EntryOptions {
   pub name: Option<String>,
   pub runtime: Option<EntryRuntime>,
   pub chunk_loading: Option<ChunkLoading>,
   pub async_chunks: Option<bool>,
   pub public_path: Option<PublicPath>,
   pub base_uri: Option<String>,
-  pub filename: Option<FilenameTemplate>,
+  #[derivative(
+    Hash = "ignore",
+    PartialEq = "ignore",
+    PartialOrd = "ignore",
+    Ord = "ignore"
+  )]
```

Comment:
@ahabhgk  help~ I'm not sure if ignored is possible here

Reply:
The Hash, PartialEq, Eq, PartialOrd, and Ord traits can potentially be removed now.
These were previously used for testing in Rust.